common/database.cpp

```diff
@@ -2158,6 +2158,28 @@ uint32 Database::GetGuildIDByCharID(uint32 character_id)
 	return atoi(row[0]);
 }
 
+uint32 Database::GetGroupIDByCharID(uint32 character_id)
+{
+	std::string query = fmt::format(
+        SQL(
+            SELECT groupid
+            FROM group_id
+            WHERE charid = '{}'
+        ),
+        character_id
+    );
```

Comment:
Formatting?

Reply:
Are you referring character_id and charid or the newlines?

Reply:
![image](https://user-images.githubusercontent.com/3319450/76174448-47175200-6175-11ea-8a18-a0a266b3f23b.png)

Closing parenthesis

Reply:
Ah, okay, will resolve.
My editor's formatting is off.
Using VSCode with TabSize 4 and not using Spaces as Tabs.